patterns/just-arrived-full-hero.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Title: WooCommerce Just Arrived Full Hero
+ * Slug: woocommerce-blocks/just-arrived-full-hero
+ * Categories: WooCommerce
+ */
+?>
+<!-- wp:group {"align":"full","style":{"spacing":{"margin":{"top":"0px","bottom":"0px"}},"elements":{"link":{"color":{"text":"var:preset|color|black"}}}},"textColor":"black","layout":{"inherit":true,"type":"constrained"}} -->
```

Comment:
What is the advantage of setting color for this block over using default color from theme? Beside, the textColor is set to a fixed color (`black`), which make this a bit harder to use on dark themes.

I think we should use the default color from theme when possible.

Reply:
Agreed. I'll remove the group and that will take care of that.